sleap/gui/app.py

```diff
@@ -1120,6 +1148,35 @@ def _update_gui_state(self):
 
         # Update menus
 
+        def _update_sessions_menu(self, frame_idx):
+            """Update the instance groups menu based on the selected frame index in the session."""
+            session = self.state.get("session")
+            if not session or frame_idx >= len(session.frame_groups):
+                return 
```

Comment:
There does not need to be a `FrameGroup` for `frame_idx`, so `frame_idx` will almost always be greater than `len(session.frame_groups)` 
```suggestion
            if session is None:
                return 
```